modules/Lending.py

```diff
@@ -256,10 +256,15 @@ def get_min_daily_rate(cur):
                 max_active_alerted[cur] = True
                 log.log('maxactive amount for ' + cur + ' set to 0, won\'t lend.')
             return False
-        cur_min_daily_rate = Decimal(coin_cfg[cur]['minrate'])
+        if exchange == 'BITFINEX' and coin_cfg[cur]['frrasmin']:
+            frr_rate = Decimal(api.get_frr(cur)) + ( Decimal(coin_cfg[cur]['frrdelta']) / 100 )
```

Comment:
Why is frrdelta added to the current frr? Wouldn't that make your loan LESS likely to be picked since your rate would be higher than FRR? Additionally, why is the frrdelta being divided by 100? If I set frrdelta to 0.0001, then it's divided by 100 to = 0.000001 which is added to the FRR of (at this time 0.0019), that gives my frr_rate = 0.001901

But don't I want 0.001899 so that I am lower than FRR?

Reply:
Hi, please use the Bitfinex interface to understand FRRdelta. The same approach is giving here. api.get.frr is already coin/100 so you need to do it for frrdelta, too. Later it will multiply with 100.